content/docs/deploy/enterprise.md

```diff
@@ -1,46 +1,109 @@
 
-In addition to the capabilities provided by open-source Pomerium, Pomerium Enterprise provides:
+Pomerium Enterprise allows you to import groups defined by your identity provider. This allows you to build stable policies that don’t require adjustments as your company changes.
 
-### Management GUI
+![Manage Namespaces in the Enterprise Console's Namespaces dashboard](./enterprise/img/manage-namespaces.gif)
 
-Pomerium Enterprise lets you view traffic and logs, define routes and policies, and organize your service access from an intuitive web interface.
+### Deployment History and Audit Logs
 
-![Overview animation of Pomerium Enterprise](./enterprise/img/console-overview.gif)
+The **Deployments** dashboard allows you to view and export change and access logs from the Enterprise Console. Within the Deployments dashboard, you can select a change to view its logs.
 
-### Programmatic API
+This way, you can review which users applied a change and when.
 
-Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice. Everything that is manageable in the Management GUI can also be driven programmatically through the API.
+![View deployments in the Enterprise Console's Deployments dashboard](./enterprise/img/deployments-dashboard.gif)
 
-### Session management
+### Directory Sync
 
-Quickly view who is logged in your infrastructure, with easy access to revoke sessions.
+Pomerium Enterprise's **Directory Sync** feature allows you to import your identity provider's directory data. After a successful sync, you can use your internal groups and teams data when building policies. ![Viewing the Identity Providers settings for a directory sync in the Enterprise Console](./enterprise/img/directory-sync-2.png)
 
-![Pomerium Enterprise Session List](./enterprise/img/console-session-list.png)
+### External Data Sources
 
-### Self-Service & Governance
+In the **External Data** dashboard, you can import, view, and manage [external data sources](/docs/integrations). After a successful sync, you can use data unique to your organization to serve as context in your authorization policies. ![Using the External Data Source Record type in a policy](./enterprise/img/external-data-as-context.gif)
 
-Easily define who can control access to what areas of your infrastructure. Our [Namespaces](/docs/capabilities/namespacing) make it easy to allow teams to self-manage access to the infrastructure they build from or depend on.
+## Pomerium Enterprise features comparison
 
-[User roles](/docs/capabilities/namespacing#rbac-for-enterprise-console-users) are granted along Namespace hierarchy, with inheritance from parents.
+| Features | Pomerium Core | Pomerium Enterprise |
+| :-- | :-- | :-- |
+| Enterprise Console | <ClearIcon /> | ![Pomerium checkmark](./enterprise/img/pomerium-checkmark.svg) |
```

Comment:
nit: I think I'm used to seeing these tables start in the other direction (with the core features first), but no strong preference from me

Reply:
The intention was to push the Enterprise features and their links to the top of the table, but I agree it does look "off" compared to what you see on most tables.